api/assets/js/main.js

```diff
@@ -0,0 +1,26 @@
+$( document ).ready(function(){
+			$('#loading').hide();
+			$('#generated-stamp').hide();
+
```

Comment:
Let's run this through a formatter once. (I should have also done the same in the previous PR)

Reply:
Sure!